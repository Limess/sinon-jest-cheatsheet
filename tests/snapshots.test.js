const React = require('react');
const ReactTestRenderer = require('react-test-renderer');

const operations = require('../src/index');
const Button = require('../src/react-component/Button');

it('should match a function', function () {
  expect(operations.add(45, 234)).toMatchSnapshot();
});

it('should match a React Component', function () {
  expect(ReactTestRenderer.create(React.createElement(Button))).toMatchSnapshot();
});
