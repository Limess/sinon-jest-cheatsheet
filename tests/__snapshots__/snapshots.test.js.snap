// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match a React Component 1`] = `
<button
  onClick={[Function]}
>
  Click me
</button>
`;

exports[`should match a function 1`] = `undefined`;
